docs/doc.md

```diff
@@ -51,7 +51,7 @@ We first define a "collation header" as an RLP list with the following values:
         period_start_prevhash: bytes32,
         parent_collation_hash: bytes32,
         tx_list_root: bytes32,
-        coinbase: address,
+        coinbase: address,                  # address chosen by creator of Shard Header
```

Comment:
Would you mind removing the comment, and add it to the explanation section under
```
Where:
...
```

Reply:
Yes, sorry I didn't realise. I'll push through changes with that fix